rust/kcl-lib/src/parsing/ast/types/mod.rs

```diff
@@ -3015,6 +3015,10 @@ impl BinaryOperator {
             Self::And | Self::Or => Associativity::Left,
         }
     }
+
+    pub fn associative(&self) -> bool {
+        matches!(self, Self::Add | Self::Mul | Self::And | Self::Or)
```

Comment:
It's a little confusing that this omits a lot of operators. Could you leave a comment explaining why?

Reply:
Are there associative operators that should be on the list?